src/main/java/kitchenpos/order/domain/TableGroup.java

```diff
@@ -44,8 +50,8 @@ public void cancleGroup() {
         this.orderTables.cancleGroup();
     }
```

Comment:
테이블 그룹 해제시 아래와 같은 검증을 해보면 좋겠네요.
`"주문 상태가 조리중이거나 식사중인 테이블의 그룹 지정은 해지할 수 없습니다."`

https://github.com/next-step/jwp-refactoring/pull/389#discussion_r775564593

```suggestion
        this.orderTables.cancelGroup();
    }
```

Reply:
TableGroupServiceTest class 안에 deleteCookingOrEatingOrderGroupTest()로 검증을 하였는데 혹시 추가 테스트가 더 필요한 것일까요??